### ISSUE TITLE
setting up comsol license 

### ISSUE THREAD

**Issue by raviapatel** [MPh-py/MPh#130]
I am using the floating license from university. I am able to launch comsol gui and clinet however while using the mph i get error. Any ideas or workaround to solve this...?
![image](https://user-images.githubusercontent.com/16664989/225968810-8171447e-ebac-44ca-8c8f-cd9593de0fa1.png)

**Comment by raviapatel** [comment]
Resloved by setting mph.option('classkit', True) before starting client

**Comment by john-hen** [comment]
To be clear, if that fixed it, then you're not actually using a "floating network license", but a "class kit license". It's a different thing, [according to Comsol](https://www.comsol.com/products/licensing). These license differences are notoriously under-documented, on Comsol's side. Which is why none of it is ever mentioned in the MPh documentation. But if more and more people are using these "unusual" licenses (from my perspective, "class kit" falls into that category), then maybe I'll have to write something on that.

Though I'd rather not. Licensing should be a thing Comsol deals with, as they're the ones making the money. I just wish class-kit licenses would work like all (or "most of") the other licenses, and not require you to call a special command. Though that's where we are.

**Comment by raviapatel** [comment]
> To be clear, if that fixed it, then you're not actually using a "floating network license", but a "class kit license". It's a different thing, [according to Comsol](https://www.comsol.com/products/licensing). These license differences are shamefully under-documented, on Comsol's side. Which is why none of it is ever mentioned in the MPh documentation. But if more and more people are using these "unusual" licenses (from my perspective, "class kit" falls into that category), then maybe I'll have to write something on that.
> 
> Though I'd rather not. Licensing should be a thing Comsol deals with, as they're the ones making the money. I just wish class-kit licenses would work like all (or "most of") the other licenses, and not require you to call a special command. But that's where we are.

Yes you are right its class kit license and indeed it remains underdocumented. I found some old issue of MPh (probably first occurance of "class kit") where this  fix was mentioned. Probably it will be nice to explain tis "class kit" option in documentation as it exist already in MPh.

**Comment by john-hen** [comment]
> I am also curious about ASL type if it works normally if this has not been tested I can try to test it next week.

Yes, please. I cannot test this myself, so if you have the opportunity it would be great if you could try and report back. If it doesn't work, you can open a new issue, like "Support academic server license" or something. I would have to know how you'd usually start the Comsol GUI and `comsolmphserver` with that license.
